lib/ansible/modules/packaging/language/pip.py

```diff
@@ -564,17 +569,20 @@ def main():
         elif rc != 0:
             _fail(module, cmd, out, err)
 
-        if state == 'absent':
-            changed = 'Successfully uninstalled' in out_pip
+        if venv_created:
```

Comment:
you can juste move this part after all processing (~L578) and keep the rest of the code intact 
```python
changed = changed or venv_created
```